cli/cmd/plugin/ui/web/tanzu-ui/src/shared/constants/defaults/aws.defaults.tsx

```diff
@@ -40,6 +112,6 @@ const AWS_DEFAULT_INSTANCE_TYPES: KeyOfStringToString = {
  * defaults map.
  * Returns default aws instance type string corresponding to selected node profile.
  */
-export function validateDefaultNodeType(nodeProfile: string): string {
+export function validateDefaultNodeType(nodeProfile: string): Array<string> {
```

Comment:
nit: I'm confused by the name of the method. Perhaps `getDefaultNodeTypes` would be clearer?